3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,217 @@
+// ________________warmup excercise : The time it will take to arrive at your destination _________________
+
+function travelInformation(speedInkmperh, destinationDistanceInkm) {
+
+    let timeToRichToTheDestination = (destinationDistanceInkm / speedInkmperh);
+    const hours = Math.trunc(timeToRichToTheDestination); //same result with const hours (~~timeToRichToTheDestination) to cuts all fractional digits
+    let minutes = (timeToRichToTheDestination - hours) * 60;
+    const rminutes = Math.ceil(minutes); // To round up the minutes
+    return ('Your travel takes ${hours} hours & ${rminutes} minutes'); // It doesn't give a result for ${hours} & ${rminutes} !!!!!
+};
+
+const travelTime = travelInformation(50, 432);
+console.log(travelTime);
+
+
+// ______ warmup excercise : how much time a tv series have taken as a percentage of an average lifespan of 80 years_______
+
+function logOutSeriesText(TVSeriestiltle, daysToWatch) {
+
+    const tookDaysOfLife = (80 * 365) / daysToWatch;
+    let percentFromlife = (daysToWatch * 100) / tookDaysOfLife;
+    let rPercentOfLife = percentFromlife.toPrecision(4); // To have just 4 digits and only 3 digits after decimal point
+    return (`${TVSeriestiltle} took ${rPercentOfLife} % of my life`);
+};
+
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 9,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 19,
+        hours: 14,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 27,
+        hours: 12,
+        minutes: 0,
+    },
+];
+
+
+for (let i = 0; i < seriesDurations.length; i++) {
+    let result = logOutSeriesText(seriesDurations[i].title, seriesDurations[i].days);
```

Comment:
Thank you.
I am still a little confused regarding my question about how I can add the 3 percentages and make a subtotal of them :).
I wanted to use a counter, however, I couldn't figure it out where to put that. Because when I want to return the results, I can return only to values. I don't know in which way I can also return the summation.

Reply:
hmm, you can just print out  `totalPercentOfLife ` no?